jax-inference-offloading/jax_inference_offloading/vllm/extension.py

```diff
@@ -51,14 +52,24 @@ def device_info(self):
     )
 
   def set_sharding(self):
-    for _, module in self.model_runner.model.named_modules():
+
+    try:
+      if "UnquantizedLinearMethod" in WEIGHT_LOADER_V2_SUPPORTED:
+        WEIGHT_LOADER_V2_SUPPORTED.remove("UnquantizedLinearMethod")
+        logger.warning("Removed UnquantizedLinearMethod from WEIGHT_LOADER_V2_SUPPORTED.")
+    except Exception as e:
+      logger.warning(f"Unable to adjust WEIGHT_LOADER_V2_SUPPORTED: {e}")
```

Comment:
Two questions:

- why would this throw an exception?
- if it does, stuff will fail down the line, so why catch it?